README.md

```diff
@@ -18,9 +18,18 @@ npm install eslint-teamcity --save-dev
 ```
 
 ## Usage
+* as onrun eslint formatter:
 ```sh
   eslint --format './node_modules/eslint-teamcity/index.js' myfiletolint.js
 ```
+* as post runnig formatter, work with standart eslint json report
```

Comment:
I think that these examples need to be clearer and have more detailed descriptions. Maybe something like: 

"Run against an existing ESLint JSON report. See (insert url) for more details on generating a JSON report".

Also, could you please link to the documentation on how to generate a JSON report? I could only find how to generate an HTML doc